src/config.ts

```diff
@@ -10,7 +10,7 @@ export const config = {
 
 export const BASE_DIR = process.cwd();
 export const OUT_DIR = path.join(BASE_DIR, "slack-archive");
-export const TOKEN_FILE = path.join(OUT_DIR, ".token");
+export const TOKEN_FILE = path.join(BASE_DIR, ".token");
```

Comment:
This is a breaking change, so I'd rather not merge it in. The `trim()` above is a great idea though!